cordapp-configuration/src/main/resources/net/corda/cordapp/cordapp-configuration.properties

```diff
@@ -14,6 +14,7 @@ Corda-InitiatedFlow-Classes=IMPLEMENTS;net.corda.v5.application.flows.ResponderF
 Corda-Subflow-Classes=IMPLEMENTS;net.corda.v5.application.flows.Subflow
 Corda-Token-Observer-Classes=IMPLEMENTS;net.corda.v5.ledger.utxo.observer.UtxoLedgerTokenStateObserver
 Corda-Ledger-Named-Query-Classes=IMPLEMENTS;net.corda.v5.ledger.utxo.query.VaultNamedQueryFactory
+Corda-Ledger-Vault-Json-Factory-Classes=IMPLEMENTS;net.corda.v5.ledger.utxo.query.json.ContractStateVaultJsonFactory
```

Comment:
I'm not 100% sure that `ContractStateVaultJsonFactory` is the right name, but I also can figure out a better name atm, lets get this merged and just give it a think and if anything better comes up we can always change it in the new few weeks.